### PR TITLE
Add `osu! deskmats` news post

### PR DESCRIPTION
**Pull request by pishifat**
- [x] banner
- [x] confirm later fulfillment things
- [x] store page (link all images to this too)

### CODE REVIEW

File: news/2024/2024-08-14-osu!-deskmats.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/twitter.png)
+
+So we expanded.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/rollout.gif)
+
+## circle clicker deskmat
+
+This cursor dance through the ~~fool moon~~ night sky is a simple omage to the osu! player's daily routine: clicking circles at mach speed atop a 100% dimmed background.
```
Comment (Walavouchey) [inline news/2024/2024-08-14-osu!-deskmats.md:21]:
```suggestion
This cursor dance through the ~~fool moon~~ night sky is a simple homage to the osu! player's daily routine: clicking circles at mach speed atop a 100% dimmed background.
```

File: news/2024/2024-08-14-osu!-deskmats.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/sakurapippi.jpg)
+
+### *everything* deskmat
+
+This one is [peppy](https://osu.ppy.sh/users/2)'s personal favorite: In [osu!'s 15th anniversary art contest](https://osu.ppy.sh/home/news/2022-09-15-anniversary-art-contest-results#peppy's-picks), [YOOKA](https://osu.ppy.sh/users/15825663) crafted an ultra-dense artwork showcasing 15 years worth of references and memes.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/everything.jpg)
+
+In the production process, we worked with YOOKA to update the artwork so it fits our wide deskmats, which led to the inclusion of even more memes that generated throughout the last couple years.
```
Comment (Walavouchey) [inline news/2024/2024-08-14-osu!-deskmats.md:55]:
```suggestion
In the production process, we worked with YOOKA to update the artwork so it fits our wide deskmats, which led to the inclusion of even more memes that were generated throughout the last couple years.
```

File: news/2024/2024-08-14-osu!-deskmats.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/stats.jpg)
+
+And most importantly, it looks really cool.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/circleclicker.jpg)
+
+We have to give credit where it's due: renowned mapper [Sing](https://osu.ppy.sh/users/3795679) is the mastermind behind this design! Send your accolades his direction.
+
+If you want to snag one of these for yourself, [**head to the store page!**](LINK)
```
Comment (Walavouchey) [inline news/2024/2024-08-14-osu!-deskmats.md:35]:
```suggestion
If you want to snag one of these for yourself, [**head to the store page**](LINK)!
```

File: news/2024/2024-08-14-osu!-deskmats.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+A couple years ago, we celebrated osu!'s 15th anniversary with limited edition deskmats, and some of you pointed out that these "deskmats" were barely big enough to fit a keyboard.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/twitter.png)
+
+So we expanded.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/rollout.gif)
+
+## circle clicker deskmat
```
Comment (Walavouchey) [inline news/2024/2024-08-14-osu!-deskmats.md:19]: the casing sure is weirdly inconsistent. it seems these are product names so i'd either prepend "The" or break the rules and make "fanart" lowercase in the next heading

File: news/2024/2024-08-14-osu!-deskmats.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+layout: post
+title: osu! deskmats
+date: 2024-08-14 15:00:00 +0000
+---
+
+You sit at your desk every day. You play osu! every day. You can see where we're going with this.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/banner.jpg)
+
+A couple years ago, we celebrated osu!'s 15th anniversary with limited edition deskmats, and some of you pointed out that these "deskmats" were barely big enough to fit a keyboard.
```
Comment (Walavouchey) [inline news/2024/2024-08-14-osu!-deskmats.md:11]: just searching around, "desk mat" with a space seems to be a lot more common (and has more search results)

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-08-14-osu!-deskmats.md (hunk 4 above)
Comment (pishifat) [reply news/2024/2024-08-14-osu!-deskmats.md:19]: fanart isn't a title in the next section though

File: news/2024/2024-08-14-osu!-deskmats.md (hunk 5 above)
Comment (pishifat) [reply news/2024/2024-08-14-osu!-deskmats.md:11]: undecided here (i like "deskmat" more but that's not very convincing lol)

File: news/2024/2024-08-14-osu!-deskmats.md (hunk 4 above)
Comment (Walavouchey) [reply news/2024/2024-08-14-osu!-deskmats.md:19]: i'm not sure what you mean

`## Fanart deskmats` -> `## fanart deskmats` was what i was suggesting (would make all headings lowercase)

but well if not then the other solution works just as well :)

Comment (pishifat) [reply news/2024/2024-08-14-osu!-deskmats.md:19]: i mean "circle clicker"/"sakura pippi"/"everything" are used as titles, so it makes sense to keep them stylized with lowercase. "Fanart deskmats" is used as a descriptive header, so i think capitalizing under normal rules makes sense, and making it lowercase wouldn't

File: news/2024/2024-08-14-osu!-deskmats.md (hunk 5 above)
Comment (pishifat) [reply news/2024/2024-08-14-osu!-deskmats.md]: going with deskmats as a \~stylistic choice\~

File: news/2024/2024-08-14-osu!-deskmats.md (hunk 4 above)
Comment (pishifat) [reply news/2024/2024-08-14-osu!-deskmats.md]: new links will hopefully make this understandable

**Review by peppy — APPROVED** [review]

**Pull request merged**